.github/labeler.yml

```diff
@@ -0,0 +1,2 @@
+documentation:
+  - docs/**/*
```

Comment:
Will this add the "documentation" label if all changes are in the docs folder or if there is at least one change in the docs folder?

Reply:
https://github.com/eslint/eslint/issues/16620 let's see where `labeler` can be used instead of a bot.

Reply:
I think we can deprecate the github bot for adding labels and start using workflows instead. This will increase more scope when someone has made changes in documentation or have added newer things in the folder.

Reply:
The GitHub bot currently adds labels based on the commit message, so "feat!" will get a "breaking" label and a "feature" label, for example. Is it possible to do that with a workflow?

Reply:
I was looking for workflows that does this but couldn't find one 😕. Everyone are doing this `feat!` and `breaking` through an github app only.

Reply:
I'll close this as it makes sense having github bot already and we don't need this workflow to detect documentation changes. Thanks for reviewing :)